libraries/alicloud_ecs_instances.rb

```diff
@@ -142,7 +142,7 @@ def fetch_instance_ram_roles(region, instance_id)
         action: "DescribeInstanceRamRole",
         params: {
           RegionId: region,
-          InstanceIds: "[\"#{opts[:instance_id]}\"]",
+          InstanceIds: "[\"#{instance_id}\"]",
```

Comment:
okay now this should work .. 
`[instance_id].to_json`

Reply:
Oh yeah good call, let me test that really quickly.